lib/template.js

```diff
@@ -507,10 +507,27 @@ module.exports = (function () {
               return Promise.map(files,
                 function (file) {
                   self.emitter.emit('msg', 'reading ' + file.fullPath);
-                  return utils.read(file.fullPath).then(
-                    function (output) {
-                      self.emitter.emit('msg', 'writing ' + file.fullPath);
-                      return utils.write(file.fullPath, output, config);
+                  return fs.readFileAsync(file.fullPath).then(
```

Comment:
we may want to update the comment here that says, "writes all files..." to just say "writes or copies"